transitions/multi_report_alerts.js

```diff
@@ -85,8 +85,8 @@ const generateMessages = (alert, phones, latestReport, countedReportsIds, newRep
 // Recipients format examples:
 // [
 //    '+254777888999',
-//    'countedReport.contact.parent.parent.contact.phone',   // returns string
-//    'countedReport.contact.parent.parent.alertRecipients', // returns string array
+//    'new_report.contact.parent.parent.contact.phone',   // returns string
```

Comment:
nope, because you specify recipients for each `new_report`, you don't have access to the whole array (it's batched for performance, so you have a chunk of the `new_reports` array, not the whole thing)